src/views/AnnualReport.vue

```diff
@@ -472,9 +469,7 @@ export default class AnnualReport extends Mixins(CommonMixin, DateMixin, FilingM
   @Getter(useBusinessStore) getLastAnnualReportDate!: string
   @Getter(useBusinessStore) getLastDirectorChangeDate!: string
   @Getter(useConfigurationStore) getPayApiUrl!: string
-  @Getter(useRootStore) getReportState!: string
   @Getter(useBusinessStore) isCoop!: boolean
-  @Getter(useRootStore) isCurrentFilingEditable!: boolean
```

Comment:
This is the old code:
```
isCurrentFilingEditable (state: RootStateIF): boolean {
  return (state.currentFilingStatus === FilingStatus.NEW || state.currentFilingStatus === FilingStatus.DRAFT)
}
```

... It was always True, so I got rid of it.

Reply:
(This was only used for Annual Report filings, and the status could only every be NEW or DRAFT.)